src/Sarif.Sarifer/SpamBackgroundAnalyzer.cs

```diff
@@ -54,6 +56,8 @@ protected override bool AnalyzeCore(Uri uri, string text, string solutionDirecto
                 return false;
             }
 
+            Trace.WriteLine($"Rules loaded: {this.rules.Count}");
```

Comment:
>Trace.WriteLine($"Rules loaded: {this.rules.Count}"); [](start = 12, length = 53)

Adding a trace to know how many rules did we load

Reply:
Move this to line 50. Replace it with a message "Analyzing {uri}..."

---
In reply to: [550914172](https://github.com/microsoft/sarif-visualstudio-extension/pull/338#discussion_r550914172) [](ancestors = 550914172)